src/CMakeLists.txt

```diff
@@ -9,12 +9,12 @@ if(IDF_TARGET STREQUAL linux)
 	idf_component_register(
 		SRCS ${COMMON_SRC}
 		INCLUDE_DIRS "." "../deps/livekit-protocol-generated"
-		REQUIRES protobuf-c esp_websocket_client peer esp-libopus)
+		REQUIRES protobuf-c esp_websocket_client peer)
 else()
 	idf_component_register(
 		SRCS ${COMMON_SRC} "wifi.cpp" "media.cpp"
 	  INCLUDE_DIRS "." "../deps/livekit-protocol-generated"
-		REQUIRES driver protobuf-c esp_wifi nvs_flash esp_websocket_client peer esp_psram esp-libopus)
+		REQUIRES driver protobuf-c esp_wifi nvs_flash esp_websocket_client peer esp_psram)
 endif()
```

Comment:
I removed the audio code just for this initial PR only to make sure that no audio encoding tasks were taking place